bool state
string co_id
---
bool success